src/main/java/org/openstreetmap/atlas/tags/MinHeightTag.java

```diff
@@ -0,0 +1,35 @@
+package org.openstreetmap.atlas.tags;
+
+import java.util.Optional;
+
+import org.openstreetmap.atlas.geography.Altitude;
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.Tag.Validation;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+import org.openstreetmap.atlas.tags.annotations.validation.DoubleValidator;
+
+/**
+ * OSM min_height tag
+ *
+ * @author ajayaswal
+ */
+@Tag(value = Validation.LENGTH, taginfo = "https://taginfo.openstreetmap.org/keys/min_height#values", osm = "https://wiki.openstreetmap.org/wiki/Key:min_height")
+public interface MinHeightTag
+{
+    @TagKey
+    String KEY = "min_height";
+
+    DoubleValidator validator = new DoubleValidator();
+
+    static Optional<Altitude> get(final Taggable taggable)
+    {
+        final Optional<String> tagValue = taggable.getTag(KEY);
+
+        if (tagValue.isPresent() && validator.isValid(tagValue.get()))
+        {
+            return Optional.of(Altitude.meters(Double.valueOf(tagValue.get())));
```

Comment:
One minor thing to consider, there's a pattern of `validateAndExtract` found in other tag classes that contain numerical values (see `LayerTag` and `MaxSpeedForwardTag`). As an optional improvement, we should consider adding a `DoubleExtractor` and following the same pattern. This also works though!

Reply:
Since the extraction code is very minimal, thinking of leaving it as is, instead of creating a new class.

Reply:
Sounds good!